### DISCUSSION TITLE
Accessibility user-scalable=0 vs user-scalable="no"

### DISCUSSION THREAD

**Discussion by raroul** [GoogleChrome/lighthouse#12448] [Q&A] [answered]
Hello !
I noticed that some sites are using the meta viewport with `user-scalable` attribute set to `0` `<meta name="viewport" content="width=device-width, initial-scale=1.0, height=device-height, minimum-scale=1.0, user-scalable=0">` and still manage to get the accessibility test of the meta viewport passing.
Is it a trick to bypass the lighthouse test or setting the `user-scalable` attribute to `0` has a different effect that setting it to `"no"` ?

**Comment by patrickhulce** [comment, marked as answer]
Thanks for asking @raroul! You're right it appears the [accessibility library that Lighthouse uses](https://github.com/dequelabs/axe-core/blob/f62f0cf18f7b69b247b0b6362cf1ae71ffbf3a1b/lib/checks/mobile/meta-viewport-scale-evaluate.js) does not follow the [spec](https://drafts.csswg.org/css-device-adapt/#user-scalable), it just looks for `user-scalable=no`.

[Related discussion in axe-core](https://github.com/dequelabs/axe-core/issues/694) already suggests this isn't really a big deal anymore because major browsers either ignore this property by default of have an easy way for users to override this setting. You're welcome to file this gap over there along with the spec link I shared that `0` is treated as `fixed`, but I'm not sure they'd prioritize it :)
